cmd/thanos/receive.go

```diff
@@ -65,10 +74,26 @@ func runReceive(
 	httpMetricsBindAddr string,
 	remoteWriteAddress string,
 	dataDir string,
+	objStoreConfig *pathOrContent,
+	externalLabelsString string,
 ) error {
 	logger = log.With(logger, "component", "receive")
 	level.Warn(logger).Log("msg", "setting up receive; the Thanos receive component is EXPERIMENTAL, it may break significantly without notice")
 
+	externalLabels := labels.Labels{}
+	labelStrings := strings.Split(externalLabelsString, ",")
```

Comment:
There is `parseLabels` function that gives you that and is used in more than one place I believe, can we reuse it? https://github.com/improbable-eng/thanos/blob/59634d99e4b7aa74d53814ed9b2401f6068e17cf/cmd/thanos/rule.go#L718

Reply:
Wonderful :+1: